scripts/zones/Halvung/npcs/Operating_Lever_A.lua

```diff
@@ -0,0 +1,30 @@
+-----------------------------------
+--  Area: Halvung
+--  NPC: Operating Lever A
+--  TODO: more than 5/6 people still need verification as no sites show this requirment?
+-----------------------------------
+local ID = require("scripts/zones/Halvung/IDs")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+end
+
+entity.onTrigger = function(player, npc)
+    if player:hasKeyItem(xi.ki.BRACELET_OF_VERVE) == false then
```

Comment:
`== false` is not needed

Reply:
I changed the logic around to reflect not using ==false.